onnxscript/optimizer/_constant_folding.py

```diff
@@ -434,25 +435,54 @@ def concat(node: ir.Node, op, state: OptimizerState) -> ReturnValue:
 @register("Dropout", version=(12, None))
 def dropout(node: ir.Node, op, state: OptimizerState) -> ReturnValue:
     """Replace a Dropout by Identity when applicable."""
-    if len(node.outputs) != 1:
-        # If output mask is requested, optimization is more complex.
-        # TODO: handle this case. But unlikely to be needed in practice.
-        return None
+
+    def optimized_dropout():
+        input = node.inputs[0]
+        output = op.Identity(input)
+        if len(node.outputs) == 1:
+            return output
+        else:
+            true_tensor = onnx.helper.make_tensor("true", onnx.TensorProto.BOOL, [1], [True])
```

Comment:
Is this IR? If so
```suggestion
            true_tensor = ir.tensor([True])
```

Reply:
Thanks. But when I look at the signature or code [here](https://github.com/microsoft/onnxscript/blob/32090a8d635b297cfd90032c00b39777415d4054/onnxscript/ir/_convenience.py#L294), it is not clear this is supported or will work?

Reply:
Good catch. We can update the signature

Reply:
This is actually covered by npt.ArrayLike

Reply:
Actually, I tried and it failed, rejecting a list. BTW, I have moved the independent parts of this PR into a separate PR: https://github.com/microsoft/onnxscript/pull/1947

Reply:
Hmm. I need to fix that then

Reply:
Resolved